src/NServiceBus.Core.Tests/Causation/AttachCausationHeadersBehaviorTests.cs

```diff
@@ -11,22 +11,23 @@
     public class AttachCausationHeadersBehaviorTests
     {
         [Test]
```

Comment:
Would be good to add a test to cover custom id generation.

Reply:
Yea, will do

Reply:
Acceptance test added